package.json

```diff
@@ -27,7 +27,7 @@
     "create-entity": "npm run typeorm entity:create -- -n "
   },
   "dependencies": {
-    "@nestjs-modules/mailer": "^1.5.1",
+    "@nestjs-modules/mailer": "^1.8.1",
```

Comment:
why did u update the version?

Reply:
I was having issues with css, I decided to try another version and it worked.

Reply:
css??? care to explain?

Reply:
The html file wasn't recognising css, either inline or external, so a post online was using using  v1.8.1, I decided to upgrade it, now it works. Before now css wasn't  working.